### ISSUE TITLE
Fix Wikiversity menu links

### ISSUE THREAD

**Issue by letompouce** [openzim/mwoffliner#1661]
The wikiversity ZIM (both [en](https://library.kiwix.org/wikiversity_en_all_maxi_2021-03/A/Wikiversity:Main_Page) and [fr](https://library.kiwix.org/wikiversity_fr_all_maxi_2021-02/A/Wikiversit%C3%A9:Accueil)) are missing a bunch of links in the sections which point to several "categories".

**Comment by kelson42** [comment]
Yes categories are not scraped. See https://github.com/openzim/mwoffliner/issues/829
